include/sfx.h

```diff
@@ -389,4 +386,13 @@ typedef enum { MONO_SOUND, STEREO_SOUND } soundMode;
 #define SE_UNK_TE3_6B8 0x6B8
 #define NA_VO_MAR_AIM_HEAD 0x87F
 
+enum Sfx {
+    SFX_HARPY_WING_FLAP = 0x601,   // vabid 0
```

Comment:
The compiler will automatically number each entry so we only need the 0x601.

Reply:
I'm also thinking instead of labeling the vabid of each sfx, to just comment when it changes. The vabids are generally in large blocks See https://gist.github.com/sozud/e7dbf3f3192d66c33b003e52d8a54355

Reply:
```
import json

with open('assets/dra/sfx.json', 'r') as f:
    data = json.load(f)

def format_output(data):
    prev_id = None
    for asset in data['asset_data']:
        id = format(asset['id'] + 0x601, 'X')
        if asset['vabid'] != prev_id:
            print(f"// vabid {asset['vabid']}")
        print(f"SFX_UNK_{id},")
        prev_id = asset['vabid']

format_output(data)
```